utils/service/util_test.go

```diff
@@ -0,0 +1,32 @@
+package service
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli/v2"
+)
+
+func TestCLIFlagsToEnvVars(t *testing.T) {
+	flags := []cli.Flag{
+		&cli.StringFlag{
+			Name:    "test",
+			EnvVars: []string{"OP_NODE_TEST_VAR"},
```

Comment:
Please change `OP_` to `KROMA_` in this file.
```suggestion
			EnvVars: []string{"KROMA_NODE_TEST_VAR"},
```